kubernetes/create-deployment.yml.erb

```diff
@@ -51,11 +51,11 @@ spec:
         volumes:
           - name: nfs-export-home
             nfs:
-              server: dataloss
+              server: homes
               path: /opt/homes/home
           - name: nfs-export-services
             nfs:
-              server: dataloss
+              server: homes
```

Comment:
May I suggest using `services` here instead? We may split homes and services hosting at some point (although likely not). That being said, the path is still under `/opt/homes` so it's a bit strange.